src/compiler/checker.ts

```diff
@@ -26339,12 +26339,14 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
                     // all co-variant inferences are subtypes of it (i.e. it isn't one of a conflicting set of candidates), it is
                     // a subtype of some contra-variant inference, and no other type parameter is constrained to this type parameter
                     // and has inferences that would conflict. Otherwise, we prefer the contra-variant inference.

```

Comment:
All of the subtype calls are gone; does this prose need to be updated to reflect that?

Reply:
fixed this, thanks for catching this